submarine-server/server-core/src/main/java/org/apache/submarine/server/manager/ModelVersionManager.java

```diff
@@ -50,14 +50,10 @@ public class ModelVersionManager {
    *
    * @return object
    */
-  public static ModelVersionManager getInstance() {
+  public static synchronized ModelVersionManager getInstance() {
```

Comment:
We should not change the position of `synchronized`, changing it on this will cause the whole method to be locked at the time of the call.